Resources/config/user_context.xml

```diff
@@ -39,5 +40,9 @@
             <argument />
             <argument />
         </service>
+
+        <service id="fos_http_cache.user_context.anonymous_request_matcher" class="FOS\HttpCacheBundle\UserContext\AnonymousRequestMatcher">
+            <argument />
```

Comment:
hm, or maybe rather use a variable for the php session id - i think tihs is already asked for in the configuration, so we should use it here too.